mkdocs_monorepo_plugin/parser.py

```diff
@@ -138,6 +137,44 @@ def read(self):
         try:
             with open(self.absNavPath, 'rb') as f:
                 self.navYaml = yaml_load(f)
+
+                # This will check if there is a `docs_dir` property on the `mkdocs.yml` file of
+                # the sub folder and scaffold the `nav` property from it
+                if self.navYaml and 'nav' not in self.navYaml:
```

Comment:
This block doesn't need to be inside the `with` block and could be moved an indent left.